django-backend/fecfiler/scha_transactions/models.py

```diff
@@ -133,5 +137,8 @@ def save(self, *args, **kwargs):
             self.transaction_id = self.generate_unique_transaction_id()
         if not self.back_reference_tran_id_number and self.parent_transaction:
             self.update_parent_related_values(self.parent_transaction)
+        if self.memo_text:
+            self.memo_text.transaction_id = self.id
```

Comment:
is this necessary if there is already a fk relation between the tables with transaction.memo_text?

Reply:
This is likely going to be helpful in the future when there are many different transaction models and tables.  This field will be universal, pointing to the id of _a_ transaction in one of the tables.  You'll be able to use the new helper function in the shared app to find the transaction instead of checking for which foreign key field is populated.